.github/workflows/tag-and-build.yml

```diff
@@ -83,12 +91,22 @@ jobs:
       run: |
         sed -i -E "s|(.*CodeFlare Operator.*\[).*(\].*releases/tag/).*(\).*)|\1${{ github.event.inputs.version }}\2${{ github.event.inputs.version }}\3|" README.md
         sed -i -E "s|(.*CodeFlare-SDK.*\[).*(\].*releases/tag/).*(\).*)|\1${{ github.event.inputs.codeflare-sdk-version }}\2${{ github.event.inputs.codeflare-sdk-version }}\3|" README.md
+        sed -i -E "s|(.*AppWrapper.*\[).*(\].*releases/tag/).*(\).*)|\1${{ github.event.inputs.appwrapper-version }}\2${{ github.event.inputs.appwrapper-version }}\3|" README.md
         sed -i -E "s|(.*KubeRay.*\[).*(\].*releases/tag/).*(\).*)|\1${{ github.event.inputs.kuberay-version }}\2${{ github.event.inputs.kuberay-version }}\3|" README.md
+        sed -i -E "s|(.*Kueue.*\[).*(\].*releases/tag/).*(\).*)|\1${{ github.event.inputs.kueue-version }}\2${{ github.event.inputs.kueue-version }}\3|" README.md
+
+    - name: Adjust AppWrapper dependency in the code
```

Comment:
I mirrored what was already done for Kuberay, but I'm not sure it's really the right thing to do.  The version number also is embedded in go.mod and go.sum and we don't make any attempt to validate that those are consistent with the versions we are setting here.

Reply:
We have the `modules` target in the Makefile that's supposed to do that for KubeRay. It could be done for Kueue as well, but I'd be inclined to have it done in a separate PR if needs be so we can merge this one ASAP.

Reply:
ok.  I'll take a look at how that works and do it as a separate PR for Kueue (and appwrapper) later.